src/autocomplete.js

```diff
@@ -51,14 +58,19 @@ export default class Autocomplete extends Component {
     onConfirm: () => {},
     confirmOnBlur: true,
     showNoOptionsFound: true,
-    required: false
+    required: false,
+    values: []
   }
 
   elementRefs = {}
 
   constructor (props) {
     super(props)
 
+    if (!props.source) {
+      props.source = createSimpleEngine(props.values)
```

Comment:
Interesting. Reassigning `props` is not recommended, however this is setting a default in a way that isn't possible using `defaultProps`.

I think this is fine.